htdocs/wp-content/themes/owc-formulieren/assets/js/editor/personal-heading/edit.js

```diff
@@ -0,0 +1,331 @@
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, MediaUpload, MediaUploadCheck, RichText } from '@wordpress/block-editor';
+import {
+	Card,
+	CardHeader,
+	CardBody,
+	Button,
+	__experimentalGrid as Grid,
+	__experimentalDivider as Divider,
```

Comment:
We maken geen gebruik van experimentele API's in de OpenPDD

https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#experimental-and-unstable-apis